drivers/telescope/lx200_10micron.cpp

```diff
@@ -233,6 +268,7 @@ bool LX200_10MICRON::ReadScopeStatus()
     }
     DEBUGFDEVICE(getDefaultName(), DBG_SCOPE, "CMD <%s> RES <%s>", cmd, data);
 
+    // TODO: check if this needs changing when satellite tracking
```

Comment:
the mount command protocol docs of 2.14.11 list only 8 fields for :Ginfo#
what doc version did you write against ?

Reply:
Hi, I am using a command protocol we got sent from 10micron upon request from the university. It should apply to mounts with 2.15.8 and above. I think with this comment I meant the output of :Gstat#, because it has states specific for satellite tracking